frontend/src/pages/admin/access/ManageAccessRequests.test.tsx

```diff
@@ -13,6 +12,7 @@ import * as API from 'constants/API';
 import { ensureGridApiHasBeenSet } from '../../../utils/testUtils';
 import ManageAccessRequests from './ManageAccessRequests';
 import { AgGridReact } from 'ag-grid-react/lib/agGridReact';
+import { act as domAct } from 'react-dom/test-utils';
```

Comment:
Is there a reason to have an alias?

Reply:
Given that there are two different act functions, (one for react-dom, one from react-test-renderer) i've been consistently calling the import from react-dom domAct so that I can differentiate them (since they are not interchangeable).